dali/operators/generic/slice/slice_attr.h

```diff
@@ -33,28 +33,37 @@
 
 namespace dali {
 
-class SliceAttr {
+class NamedSliceAttr {
  public:
-  explicit inline SliceAttr(const OpSpec &spec)
+  explicit inline NamedSliceAttr(const OpSpec &spec,
+                                 const char* start_name = "start",
```

Comment:
Do you expect to rename this arguments?

Reply:
Yes, "start" in the context of numpy reader doesn't make much sense. In the reader I call it "roi_start"

Reply:
OK